src/Security/Authentication/JwtBearer/ref/Microsoft.AspNetCore.Authentication.JwtBearer.csproj

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
I think CodeCheck is complaining about the extra lines in this file. Revert.

Reply:
done